### ISSUE TITLE
RFQ: supporting RISC-V architecture based Milk-V Duo

### ISSUE THREAD

**Issue by milkrisc-v** [OpenIPC/firmware#1195]
Milk-V Duo is an ultra-compact embedded development platform based on the CV1800B chip with two RISC-V Cores, one C906 running Linux and one C906 running RTOS. 

- Integrated CVITEK TPU for smart detection.
- Supports H.264/H.265 video encoding, up to 2880x1620@20fps
- Features a 16-pin FPC interface for 2-lane MIPI camera input
- Up to 4M Pixel ISP support

**Comment by milkrisc-v** [comment]
We(Milk-V) are very happy to donate Duos and Cameras to have OpenIPC support. If the team is interested, please contact me at: v@milkv.io .

**Comment by FlyRouter** [comment]
Good afternoon
Thank you very much for your suggestion.
We as a team will think about it, read the documentation and I will contact you a little later.

**Comment by piotr-defendeye** [comment]
Yes! Support for MilkV would be amazing! Really looking forward to it!

**Comment by piotr-defendeye** [comment]
Hi! Any success with that? :D
